libs/signed_integers/src/i128.sw

```diff
@@ -107,20 +123,20 @@ impl core::ops::Divide for I128 {
             || self.underlying == Self::indent())
             && divisor.underlying > Self::indent()
         {
-            res = Self::from((self.underlying - Self::indent()) / (divisor.underlying - Self::indent()) + Self::indent());
+            res = Self::from_uint((self.underlying - Self::indent()) / (divisor.underlying - Self::indent()) + Self::indent());
```

Comment:
`from` indents the value, `from_uint` uses value as is. `From` exists as a convenience function, so that developers don't indent their numbers all the rime by hand